docarray/array/storage/qdrant/find.py

```diff
@@ -0,0 +1,92 @@
+from typing import (
+    Union,
+    TYPE_CHECKING,
+    TypeVar,
+    Sequence,
+    List,
+)
+
+from .helper import QdrantStorageHelper
+from .... import Document, DocumentArray
+from ....math import ndarray
+from ....score import NamedScore
+from qdrant_openapi_client.models.models import Distance
+
+if TYPE_CHECKING:
+    import tensorflow
+    import torch
+    import numpy as np
+    from qdrant_client import QdrantClient
+
+    QdrantArrayType = TypeVar(
+        'QdrantArrayType',
+        np.ndarray,
+        tensorflow.Tensor,
+        torch.Tensor,
+        Sequence[float],
+    )
+
+
+class FindMixin:
+    @property
```

Comment:
I think if u declare as abstract method it will not allow u to instantiate the class and will fail faster.

Reply:
done